src/icons/Attention.tsx

```diff
@@ -2,8 +2,16 @@ import * as React from 'react'
 import Svg, { Path } from 'react-native-svg'
 import Colors from 'src/styles/colors'
 
-const AttentionIcon = ({ color = Colors.warningDark }) => (
-  <Svg width={16} height={16} fill="none">
+const AttentionIcon = ({
+  color = Colors.warningDark,
```

Comment:
I wonder if we should use a limited set of colors to ensure consistency?

Reply:
my vote is no unless this request came from design (i.e. we've committed to using a set in future designs) - i think deriving what the limited set should be based on previous usage will not be reliable for predicting future designs and we'll end up augmenting the set, defeating the purpose. the app `colors` enum has shrunk recently and ensures some level of consistency already, so i'm not sure that i see the need to define a subset of the enum